Framework/docs/tutorial/time-server.md

```diff
@@ -191,7 +209,7 @@ The other way of communicating with server are WebSockets - bi-directional commu
 Create an instance of the WebSocket client. Then you can either send or listen to messages.
 The following `this._prepareWebSocket()` method (note that by convention all method names prepended with `_` are private) listens to two events:
  -  `authed` - notifies that client has successfully authorized by the server (automatically generated by server)
- - `server-date` - custom message that includes server's time (as defined in the [Explaining server side](#explaining-server-side) section - look for `wsServer.bind`)
+ - `server-date` - custom message that includes server's time (as defined in the [Server side explained](#server-side-explained) section - look for `wsServer.bind`)
```

Comment:
It looks like the correct section to link to is:
https://github.com/AliceO2Group/WebUi/blob/dev/Framework/docs/tutorial/time-server.md#server-side-explained

The previous anchor `#explaining-server-side` does not exist.